### PR TITLE
Exclude hidden sheets

### PR DESCRIPTION
**Pull request by dharness**
- Add an option to `.open()` to exclude sheets marked as "hidden" in excel from being loaded
- Add new test file with hidden sheets

### Example:

`XlsxReader.open(filepath, exclude_hidden_sheets?: true)`

### CODE REVIEW

File: test/xlsx_reader/parsers/worksheet_parser_test.exs [hunk 1]
```diff
@@ -137,7 +137,6 @@ defmodule XlsxReader.Parsers.WorksheetParserTest do
     assert expected == sheets
   end
 
-  @tag runnable: true
```
Comment (dharness) [inline test/xlsx_reader/parsers/worksheet_parser_test.exs:140]: I accidentally left this here in an earlier PR.

**Comment by xavier** [comment]
Thanks! 👍

**Comment by dharness** [comment]
@xavier do you have an idea of when this will make its way into a release?

**Comment by xavier** [comment]
@dharness The release has been published. Thanks again for your contributions and your patience.

**Pull request merged**